scripts/cmake-presets/ci-ubuntu-github.json

```diff
@@ -126,6 +127,15 @@
         "CELERITAS_UNITS": "CLHEP"
       }
     },
+    {
+      "name": "reldeb-vecgeom-covfie",
+      "description": "FINAL: Covfie field",
+      "inherits": ["type-reldeb", ".vecgeom", "spack"],
+      "cacheVariables": {
+        "CELERITAS_USE_Covfie": {"type": "BOOL", "value": "ON"},
+        "CMAKE_CXX_STANDARD": "20"
+      }
+    },
```

Comment:
If we don't have a special build for covfie, we could change the preset in  build-spack.yaml to differentiate between the cxxstd compiler, or move the `CELERITAS_USE_Covfie` into the spack.yaml itself (e.g., `cmake --preset=${CMAKE_PRESET} --log-level=VERBOSE -D CELERITAS_USE_Covfie=${{env.CXXSTD == '20'  && 'ON' || 'OFF'}}`), or we could leave it as automatic (unspecified).

Reply:
mmh, not sure what do you mean by leaving it automatic (unspecified). Wouldn't it always be disabled then?

Reply:
No, the auto-detection would (should) enable it if it's found, or disable if not. It's a bit risky since if it suddenly stopped finding it, we'd have no way to know.